index.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 
 import Bluebird from "bluebird"
-import newrelic from "artsy-newrelic"
+import "artsy-newrelic"
```

Comment:
importing the module will still eval all of the usual setup for crashes, https://github.com/artsy/artsy-newrelic/blob/master/index.js#L4-L16 but not using the default export will skip the [browser section](https://github.com/artsy/artsy-newrelic/blob/master/index.js#L18-L22).